src/internal/m365/collection/exchange/events_container_cache.go

```diff
@@ -70,22 +71,40 @@ func (ecc *eventContainerCache) Populate(
 		return clues.Wrap(err, "initializing")
 	}
 
-	err := ecc.enumer.EnumerateContainers(
+	el := errs.Local()
+
+	containers, err := ecc.enumer.EnumerateContainers(
 		ctx,
 		ecc.userID,
 		"",
-		false,
-		ecc.addFolder,
-		errs)
+		false)
 	if err != nil {
 		return clues.Wrap(err, "enumerating containers")
 	}
 
+	for _, c := range containers {
+		if el.Failure() != nil {
+			return el.Failure()
+		}
+
+		gncf := graph.NewCacheFolder(
+			api.CalendarDisplayable{Calendarable: c},
```

Comment:
Honestly, I'm not sure the `CalendarDisplayable` is an wrapper we need to support in the first place.  Its design is predicated on interfaces which we refactored away from some time ago.  I haven't done thorough evaluation, but I'd guess we can safely toss the wrapper entirely now that we're processing items within discrete handlers instead of running them all against code that expects normalized item interfaces.